include/xrpl/protocol/TxMeta.h

```diff
@@ -161,7 +161,8 @@ class TxMeta
     getGasUsed() const
     {
         XRPL_ASSERT(
-            hasGasUsed(), "ripple::TxMeta::getGasUsed : non-null batch id");
+            hasGasUsed(),
+            "ripple::TxMeta::getGasUsed : non-null gas used field");
         return *gasUsed_;
```

Comment:
Assert doesn't work on release build. Please add check and return some default value

Reply:
There is a check every time it's called anyways